modules/file/file.go

```diff
@@ -132,8 +142,11 @@ func (s *search) makeChecks() (err error) {
 			err = fmt.Errorf("makeChecks() -> %v", e)
 		}
 	}()
+	if s.Options.Debug == "print" {
+		debug = true
+	}
 	if s.Options.MaxDepth == 0 {
-		s.Options.MaxDepth = 10000
+		s.Options.MaxDepth = 1000
 	}
 	if s.Options.MatchLimit == 0 {
 		s.Options.MatchLimit = 1000
```

Comment:
just realized we had that here, even thus its not part of the patch: im wondering if that could lead to confusion as 1000 is a great number of sub dirs, but a potentially low number of matches /offtopic